### ISSUE TITLE
Unable to login.

### ISSUE THREAD

**Issue by beedell-roke** [nextcloud/talk-android#3136]
### Steps to reproduce

1. Install
2. Login using the account selector

### Expected behaviour

The account should authenticate.

### Actual behaviour

```
The web page at http://null/index.php/login/flow could not be loaded because:

net::ERR_NAME_NOT_RESOLVED
```

![image](https://github.com/nextcloud/talk-android/assets/42837531/c76aa471-1ec3-4436-b1c2-46240b2cadc8)

### Device brand and model

Fairphone 4

### Android version

12

### Nextcloud Talk app version

17.0.0

### Custom Signaling server configured

None

### Custom TURN server configured

None

### Custom STUN server configured

None

**Comment by AndyScherzinger** [comment]
No reproduction information was provided and there has been no activity since 2023; the login flow has been rewritten since (#5519). Closing as stale — please open a fresh report if login still fails on 25.x.

**Comment by beedell-roke** [comment]
@AndyScherzinger, indeed, it doesn't reproduce with the most recent `versionCode` that is available via `com.android.vending`. Thanks.
